static/js/components/ConfirmDeletion.js

```diff
@@ -18,32 +20,34 @@ export default class ConfirmDeletion extends React.Component {
   };
 
   render () {
-    const { close, open, confirmText } = this.props;
+    const { close, open, itemText } = this.props;
     let actions = [
       <Button
         type='button'
         key='close'
-        className="cancel-button"
+        className="secondary-button cancel-button"
         onClick={close}>
-        No
+        Cancel
       </Button>,
       <Button
         key='delete'
         type='button'
-        className="delete-button"
+        className="primary-button delete-button"
         onClick={this.deleteAndClose}>
-        Yes
+        Delete
       </Button>
     ];
     return (
       <Dialog
-        className={"deletion-confirmation"}
+        title={dialogTitle(itemText)}
+        className="deletion-confirmation-dialog-wrapper"
+        titleClassName="dialog-title"
+        contentClassName="dialog deletion-confirmation-dialog"
```

Comment:
I can't find `deletion-confirmation-dialog` anywhere. Is it left here for consistency with the other `Dialog` components?